.github/pull_request_template.md

```diff
@@ -0,0 +1,62 @@
+# Which issue does this PR close?
+
+<!--
+Thanks for opening a pull request!
+If this is your first pull request you can find detailed information on how 
+to contribute here:
+  * [New Contributor's Guide](https://arrow.apache.org/docs/dev/developers/guide/step_by_step/pr_lifecycle.html#reviews-and-merge-of-the-pull-request)
+  * [Contributing Overview](https://arrow.apache.org/docs/dev/developers/overview.html)
+
+
+If this is not a [minor PR](https://github.com/apache/arrow/blob/master/CONTRIBUTING.md#Minor-Fixes). Could you open an issue for this pull request on GitHub? https://github.com/apache/arrow/issues/new/choose
+
+Opening GitHub issues ahead of time contributes to the [Openness](http://theapacheway.com/open/#:~:text=Openness%20allows%20new%20users%20the,must%20happen%20in%20the%20open.) of the Apache Arrow project.
+
+Then could you also rename the pull request title in the following format?
+
+    GH-${GITHUB_ISSUE_ID}: [${COMPONENT}] ${SUMMARY}
+
+or
+
+    MINOR: [${COMPONENT}] ${SUMMARY}
+
+In the case of old issues on JIRA the title also supports:
+
+    ARROW-${JIRA_ISSUE_ID}: [${COMPONENT}] ${SUMMARY}
+    PARQUET-${JIRA_ISSUE_ID}: [${COMPONENT}] ${SUMMARY}
+
+-->
+Closes #
+
+# Rationale for this change
```

Comment:
Seeing some first PRs with this, shall we use a smaller header? (eg `###` instead of `#`)

Reply:
Current header does looks too big IMO.